data_steward/bq_utils.py

```diff
@@ -15,6 +15,7 @@
 # Project imports
 import app_identity
 import common
+from gcloud.gcs import StorageClient
 import gcs_utils
```

Comment:
Do you think this can be removed as well? There is one dependency which might be simple to switch to the new storage client.

Reply:
Yes,  I can definitely chop the last `gcs_utils` reference out.  Sometimes these changes get "out of control" and are difficult for the reviewer, so I try to keep on the ticket as best I can for the review's sake.